pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -78,7 +78,7 @@ func (d *ChainDescriber) MakeGraph() (osgraph.Graph, error) {
 // image stream tag (name:tag) in namespace. Namespace is needed here
 // because image stream tags with the same name can be found across
 // different namespaces.
-func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag) (string, error) {
+func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, buildInputEdgeKinds []string) (string, error) {
```

Comment:
How would you feel about passing a flag or having different describe methods or something to indicate intent instead of leaking knowledge of the flag library to callers?